boards/arm/npcx7m6fb_evb/npcx7m6fb_evb_defconfig

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
Enable ARM_MPU here if the SoC supports it (you probably need to select CPU_HAS_ARM_MPU in the soc definition)

Reply:
@MulinChao , Is this something that we need to do or not?